browser/extensions/brave_wallet_apitest.cc

```diff
@@ -79,5 +79,32 @@ IN_PROC_BROWSER_TEST_F(BraveWalletExtensionApiTest,
   ASSERT_TRUE(catcher.GetNextResult()) << message_;
 }
 
+IN_PROC_BROWSER_TEST_F(BraveWalletExtensionApiTest,
+    BraveShieldsDappDetectionWhenEnabled) {
+  ResultCatcher catcher;
+  const Extension* extension =
+    LoadExtension(extension_dir_.AppendASCII("braveShieldsWithWallet"));
+
+    ASSERT_TRUE(browsertest_util::ExecuteScriptInBackgroundPageNoWait(
+      browser()->profile(), brave_extension_id,
+      "testEnabled()"));
+  ASSERT_TRUE(extension);
+  ASSERT_TRUE(catcher.GetNextResult()) << message_;
+}
+
+IN_PROC_BROWSER_TEST_F(BraveWalletExtensionApiTest,
+    BraveShieldsNoDappDetectionWhenNotEnabled) {
+  GetPrefs()->SetBoolean(kBraveWalletEnabled, false);
+  ResultCatcher catcher;
+  const Extension* extension =
+    LoadExtension(extension_dir_.AppendASCII("braveShieldsWithWallet"));
+
+    ASSERT_TRUE(browsertest_util::ExecuteScriptInBackgroundPageNoWait(
+      browser()->profile(), brave_extension_id,
+      "testNotEnabled()"));
```

Comment:
Nit: Seems like the indentation may be off here.

Reply:
Strangely lint did pass and passes on CI 🤷‍♂ 
Updated though in any case since alignment did seem off there.